### DISCUSSION TITLE
packages.termux.org is down?

### DISCUSSION THREAD

**Discussion by finagolfin** [termux/termux-packages#10976] [General]
It redirects to packages.termux.dev which is down. Is this planned? packages-cf.termux.dev still works.

**Comment by agnostic-apollo** [comment]
The redirection was announced yesterday on matrix dev channel. Yes, packages.termux.dev is having random ssl issues. I enabled debug logging few hours ago and restarted server in hopes to get a clue why it fails. Will check it.

https://matrix.to/#/!rrHhKvLYtBVDOISphu:gitter.im/$lIbpGIe9_FP8F6d0T_pBBRGpGn4gMThDijWjofRzOwI?via=matrix.org&via=gitter.im&via=aria-net.org

**Reply by Grimler91** [reply]
After some testing and discussion with agnostic-apollo it seems to be an issue with fosshost's proxy, possibly similar/related to issue in https://github.com/termux/termux-app/issues/2650. Resolving to 216.18.208.171 and 216.18.208.172 always fails while 170, 173 and 174 always works:

```
$ curl --resolve "packages.termux.dev:443:216.18.208.171"  https://packages.termux.dev
curl: (35) OpenSSL SSL_connect: SSL_ERROR_SYSCALL in connection to packages.termux.dev:443
$ curl --resolve "packages.termux.dev:443:216.18.208.170"  https://packages.termux.dev
# ... success
```

I have opened a ticket at support.fossho.st.

**Reply by finagolfin** [reply]
May be a broader issue going around as [my github action CI wasn't able to connect to their CI cache either 11 hours ago](https://github.com/buttaface/swift-android-sdk/runs/6858264993?check_suite_focus=true#step:3:6).

**Reply by Grimler91** [reply]
"they/their" being fosshost, i.e. swift also uses fosshost?

**Reply by Grimler91** [reply]
Or you mean that github CI failed to connect to github's CI cache?

**Reply by finagolfin** [reply]
No, that is a problem with github itself, [resolved now](https://www.githubstatus.com/incidents/6r4sms3hcj3x), though they don't say if it was internal or if they were relying on some external service. I was assuming that both fosshost and github were using some common external proxy, considering they both went down at the same time, but just speculating.

**Reply by agnostic-apollo** [reply]
FYI, uploads to our fosshost server would have failed around I think 5ish hrs ago too since the error.log file grew to 3.6gb and machine ran out of space. We are low on disk space. Some space is also being consumed due to recent android 5/6 repo hosting.

**Reply by thunder-coding** [reply]
> We are low on disk space. Some space is also being consumed due to recent android 5/6 repo hosting.

You seem to be ignoring the fact that a massive 34GB of disk space is used by OS installation itself!

**Reply by agnostic-apollo** [reply]
I didn't really check what is actually consuming what space and if some can be freed. But space was critically low regardless of reason.

**Reply by Grimler91** [reply]
I've changed nginx settings, we had `access_log on;` for some reason, causing massive logs in /usr/share/nginx/. Thanks @thunder-coding for investigating

**Reply by thunder-coding** [reply]
Well, I could have found the problem yesterday itself (even before this discussion was created). I was playing around on the server and found that /usr was consuming 37G of storage. I assumed that it was due to package bloat and therefore simply ignored it (didn't care to even check the subfolders in /usr).

**Reply by agnostic-apollo** [reply]
>we had access_log on; for some reason

Likely done by fornwall/xeffyr.

>The [primary FossHost server](https://packages.termux.dev/) which serves packages records a log of remote IP, access time and what is being accessed.

https://termux.dev/privacy-policy

It should stay on with some kind of log rotation (1GB maybe), malicious users and all.
